### ISSUE TITLE
Compatibility with julia 1.0

### ISSUE THREAD

**Issue by jebej** [JuliaSIMD/LoopVectorization.jl#27]
It seems that the project files requires julia v1.3. Could that bound be lowered, or are you using certain new features?

**Comment by chriselrod** [comment]
I get this on 1.0:
```julia
ERROR: LoadError: syntax: SSAValue objects should not occur in an AST
```
I tried fixing the obvious places, but it didn't help.

**Comment by jebej** [comment]
Tests pass for me on v1.1.1, which is the version I'm using until the new threading capabilities stabilize.

**Comment by chriselrod** [comment]
I just confirmed that tests pass on Julia 1.1 locally. I updated the Julia version requirement to 1.1 and set Travis to test it. I'll push it and release a new version fairly soon.

**Comment by chriselrod** [comment]
Version 0.6 was just released and sets the minimum Julia version to 1.1.
I'll leave this open as tests don't pass on 1.0.

**Comment by chriselrod** [comment]
I have tests passing locally on v1, but it takes about 3 times longer than tests on 1.5.

That means tests on v1.0 will very likely take well over an hour to run on Travis, and thus timeout.

However, v1.0 requires adding a `compat` dependency and an additional roughly 20 lines of code to support it which would then be uncovered by tests. Not sure if it's worth merging.
